docs/metrics/selfcheckgpt.md

```diff
@@ -0,0 +1,62 @@
+# SelfCheckGPT
```

Comment:
I would like to suggest different structure for this doc if that's ok. Instead of centering around the selfcheckgpt tool itself, I want it to focus on consistency score (and we can call it SelfCheck Consistency Score). And only include the three methods that were meaningful and usable.   

And in the Implementation Details section, we can link SelfCheckGPT API for SelfCheckBertScore and SelfCheckNGram. And I would expect to see an example on how you can compute these metrics and what the expected output is in each section:

Proposed Structure:
```
# SelfCheck Consistency Score
## Implementation Details
### NGram-based Consistency Score
### BERTScore-based Consistency Score
### LLM Prompt-based Consistency Score
## Limitations and Advantages
```